frontend/packages/knative-plugin/src/components/overview/EventPubSubSubscribers.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { Button, Grid, GridItem } from '@patternfly/react-core';
+import { AngleDownIcon, AngleRightIcon } from '@patternfly/react-icons';
+import { referenceFor } from '@console/internal/module/k8s';
+import { SidebarSectionHeading, ResourceLink } from '@console/internal/components/utils';
+import { Subscriber } from '../../topology/knative-topology-utils';
+import FilterTable from './FilterTable';
+import './EventPubSubSubscribers.scss';
+
+type EventPubSubSubscribersProps = {
+  subscribers?: Subscriber[];
+};
+
+const EventPubSubSubscribers: React.FC<EventPubSubSubscribersProps> = ({ subscribers }) => {
+  const [expanded, setExpanded] = React.useState('');
+
+  const expandButton = (currentSelection, onClick) => {
```

Comment:
Can we put it outside of `EventPubSubSubscribers` component?

Reply:
Moved out to a new component